### PR TITLE
move svg into component to be reusable

### LINKED ISSUE #6: Find a reusable icon solution
At this moment all icons are part of the html as SVG. This is not very reusable and scalable.
Come with a solution, component based, that allows for reusability of icons.

External libraries can be used, they need to be open source and support tree shaking
Fonts can also be used

### PR DESCRIPTION
**Pull request by Nurintaaan**
Hello, I've move some several svg files into Vue Component based on this issue https://github.com/kyma-incubator/github-dashboard/issues/6 . I hope it can be helpful. Feel free to comment at this pull request!

### CODE REVIEW

File: src/components/Icon/LogoIcon.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M201.694 387.105C231.686 417.098 280.312 417.098 310.305 387.105C325.301 372.109 332.8 352.456 332.8 332.8C332.8 313.144 325.301 293.491 310.305 278.495C295.309 263.498 288 256 275.2 230.4C256 243.2 243.201 320 243.201 345.6C201.694 345.6 179.2 332.8 179.2 332.8C179.2 352.456 186.698 372.109 201.694 387.105Z"
+      fill="white"
+    />
+  </svg>
+</template>
```
Comment (valentinvieriu) [inline src/components/Icon/LogoIcon.vue]: Can we add the export name to all the icons
```javascript

<template>
...
</template>

<script>
export default {
  name: 'LogoIcon'
}
</script>
```

**Comment by valentinvieriu** [comment]
Thank you @Nurintaaan for your contribution, I have 2 small requests:
- One adding the exports to all Icon Components,
- Rename the Icon folder to `Icons` as we do use plural in the other folder files

**Comment by valentinvieriu** [comment]
As this pull request solves the issue and splits the SVG's in components, would be great to have something in [thisDirection](https://github.com/bytedance/IconPark) in the future.

**Comment by Nurintaaan** [comment]
All right I've been update based on your comment

**Comment by valentinvieriu** [comment]
Thank you @Nurintaaan for your contribution! If you are part of Hacktoberfest can you confirm if this pull request is considered?
We had a complain from another contributor, and want to make sure it worked for you.

**Comment by Nurintaaan** [comment]
Urwelcome @valentinvieriu :+1: . Yes it is, it works for me. Thanks a lot :bowing_woman:

**Pull request merged**